docs/src/_reference/command-line-interface.md

```diff
@@ -9,6 +9,17 @@ Meltano provides a command line interface (CLI) that makes it easy to manage you
 To quickly find the `meltano` subcommand you're looking for, use the Table of Contents in the sidebar.
 For a better understanding of command line documentation syntax, the [docopt](http://docopt.org/) standard is useful.
 
+## Global Options
+
+The following options are available for all `meltano` subcommands:
+
+* `--log-config` - Path to a logging configuration file. See [Logging](/guide/logging) for more information.
+* `--log-level`: Set the log level for the command. Valid values are `debug`, `info`, `warning`, `error`, and `critical`.
+
+## Environment Variables
+
```

Comment:
@edgarrmondragon this feels like it deserves an entry on https://deploy-preview-6785--meltano.netlify.app/reference/settings#meltano-cli ?